### PR TITLE
allow application configurable option to skip middleware cookie support

### PR DESCRIPTION
**Pull request by grossjo**
as well setup for future configurable options

### CODE REVIEW

**Comment by grossjo** [comment]
Let me know if this interests you.
My application doesn't have support for cookies from clients, so the middleware is pointless for me.

It also, opens up the ability to pass in procs, in the config, as I saw you had a todo around that.

File: lib/makara/railtie.rb [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+
+    initializer "makara.configure" do |app|
+      Makara.configure do |config|
+        config.skip_middleware = false
+        if app.config.makara[:skip_middleware]
+          config.skip_middleware = app.config.makara[:skip_middleware] 
+        end
+      end
+
+      app.config.middleware.insert_before "::Rails::Rack::Logger", "Makara::Middleware"
```
Comment (jeremy) [inline lib/makara/railtie.rb:14]: Why install the middleware at all if we're going to skip it? Can we omit it entirely?

**Comment by jeremy** [comment]
(Alternatively, you can `config.middleware.delete ::Makara::Middleware` in an initializer.)

**Comment by bleonard** [comment]
I realize the response time is not so great, but let me know if this is needed (rather than just omitting it).

**Comment by bleonard** [comment]
thanks for your interest @grossjo - let me know how it is working for you.

**Pull request closed without merging**